Library/Homebrew/extend/predicable.rb

```diff
@@ -0,0 +1,9 @@
+module Predicable
+  def attr_predicate(*attrs)
+    attrs.each do |attr|
+      define_method attr do
+        instance_variable_get("@#{attr.to_s.sub(/\?$/, "")}") == true
```

Comment:
I think this should take names without the `?` so it's consistent with `attr_writer`.